src/mongoc/mongoc-version.h.in

```diff
@@ -91,5 +91,34 @@
          (MONGOC_MAJOR_VERSION == (major) && MONGOC_MINOR_VERSION == (minor) && \
           MONGOC_MICRO_VERSION >= (micro)))
 
+/**
+ * mongoc_get_major_version:
+ *
+ * Helper function to return the runtime minor version of the library.
```

Comment:
it's our style in the C driver to put comment blocks on function definitions, not the declarations.